app/src/main/java/com/example/numberbaseball/MainActivity.kt

```diff
@@ -14,14 +15,32 @@ class MainActivity : BaseActivity(R.layout.activity_main) {
     private fun initView() {
         // 버튼 클릭 시 숫자를 랜덤하게 보여줌
         btn_get_random_number.setOnClickListener {
-            tv_random_number.text = getRandomNumber()
+            tv_random_number.text = setBaseballNumber()
+
         }
     }
 
     /**
-     * Single-expression functions
-     * (n..m).random 은 n 부터 m 까지의 랜덤한 숫자를 반환
-     * 따라서 1~9 사이의 랜덤숫자를 반환
+     * 숫자 야구때 사용할 번호 세팅
      */
-    private fun getRandomNumber(): String = (1..9).random().toString()
+    private fun setBaseballNumber(): String {
+
+        val numberMap = mutableMapOf<Int, Int>()
+
+        while (true) {
+            // (n..m).random 은 1 부터 9 까지의 랜덤한 숫자를 반환
+            val randomTemp = (1..9).random()
+            numberMap[randomTemp] = randomTemp
+
+            Log.d("MainActivity", "numberMap 데이터 : " + numberMap.values)
```

Comment:
이와 같이 사용이 가능합니다.
```suggestion
            Log.d("MainActivity", "numberMap 데이터 :  ${numberMap.values}")
```
또한, Timber 를 사용하시면, GOEAT에 적용된 Logger와 같이 사용할 수 있습니다.

Reply:
[38600e4] Timber 추가 및 적용했습니다.